pkg/php/wordpress/parse.go

```diff
@@ -0,0 +1,65 @@
+package wordpress
+
+import (
+	"io"
+	"regexp"
+
+	"github.com/aquasecurity/go-dep-parser/pkg/types"
+	"golang.org/x/xerrors"
+)
+
+type lockFile struct {
+	Packages []packageInfo
+}
+type packageInfo struct {
+	Name    string
+	Version string
+}
+
+var (
+	wpVersionRegex = regexp.MustCompile(`(\$wp_version\s*=\s*['"])(.*)(['"]\s*;)`)
+	commentRegex   = regexp.MustCompile(`(/\*([^*]|[\r\n]|(\*+([^*/]|[\r\n])))*\*+/)|(//.*)`)
+)
+
+func Parse(r io.Reader) (lib types.Library, err error) {
+
+	// If wordpress file, open file and
+	// find line with content
+	// $wp_version = '<WORDPRESS_VERSION>';
+	content, err := readAll(r)
+	if err != nil {
+		return lib, xerrors.Errorf("decode error: %w", err)
+	}
+
+	contentStr := string(commentRegex.ReplaceAll(content, []byte{}))
+
+	matches := wpVersionRegex.FindAllStringSubmatch(contentStr, 3)
```

Comment:
I think we can process line by line and simplify regular expressions.

```suggestion
	var version string
	scanner := bufio.NewScanner(r)
	for scanner.Scan() {
		line := scanner.Text()
		
		// It might include $wp_version_something
		if strings.HasPrefix(line, "$wp_version") {
			continue
		}
	
		ss := strings.Split(line, "=")
		if len(ss) != 2 || strings.TrimSpace(ss[0]) != "$wp_version" {
			continue
		}
		version = strings.Trim(ss[1], `'";`)
		break
	}
	if err := scanner.Err(); err != nil {
		return types.Library{}, xerrors.New("version.php could not be parsed")
	}
```